packages/client/tests/functional/raw-queries/typed-results/tests.ts

```diff
@@ -207,8 +207,9 @@ testMatrix.setupTestSuite(
           // PostgresError { code: \"22003\", message: \"value \\\"-18428729675200069634\\\" is out of range for type bigint\", severity: \"ERROR\", detail: None, column: None, hint: None }
           await expect(create).rejects.toThrow(`is out of range for type bigint`)
         } else if (driverAdapter === 'js_planetscale') {
+          await expect(create).rejects.toThrow(`Value out of range for the type.`)
           await expect(create).rejects.toThrow(
-            `Value out of range for the type. rpc error: code = FailedPrecondition desc = Out of range value for column 'bInt' at row 1`,
+            `rpc error: code = FailedPrecondition desc = Out of range value for column 'bInt' at row 1`,
```

Comment:
The original message is split into two, as there is something like "target: test-0000-00000000.0.primary: vttablet:" in between (as a consequence of https://github.com/prisma/prisma/pull/23198).